cfgov/jinja2/v1/_includes/organisms/filter-filterable-list.html

```diff
@@ -0,0 +1,159 @@
+{# ==========================================================================
+
+   filter_filterable_list.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create an expandable of filters when given:
+
+   settings:            Array of objects describing a form structure.
+                        Example nested form:
+                        {
+                            'type': 'form',
```

Comment:
I think this is awesome but wish we could be closer to DOM. It's a little confusing because you are using `data` for multiple purposes. Could you use `children, attrs, data, lables, tags` etc which would be closer to DOM? I am thinking as though we are taking HTML and converting it to JSON or a python dictionary in this case.